thundra/thundra_agent.py

```diff
@@ -104,6 +105,21 @@ def wrapper(event, context):
                             self.start_debugger_tracing()
 
                         response = original_func(event, context)
+                        try:
+                            if config.is_response_trace_injection_enabled():
+                                trace_link = str(uuid.uuid4())
+                                self.plugin_context['response_trace_link'] = trace_link
+                                step = 0
```

Comment:
I think, step logic should not be here because it is specific to AWS Step Functions and it should not be part of common logic here